WalletWasabi.Gui/Controls/TransactionDetails/Views/TransactionDetails.xaml

```diff
@@ -0,0 +1,43 @@
+﻿<UserControl xmlns="https://github.com/avaloniaui"
+  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+  xmlns:i="clr-namespace:Avalonia.Xaml.Interactivity;assembly=Avalonia.Xaml.Interactivity"
+  xmlns:behaviors="clr-namespace:WalletWasabi.Gui.Behaviors;assembly=WalletWasabi.Gui"
+  xmlns:local="clr-namespace:WalletWasabi.Gui.Controls.TransactionDetails;assembly=WalletWasabi.Gui"
+  xmlns:controls="clr-namespace:WalletWasabi.Gui.Controls;assembly=WalletWasabi.Gui"
+  xmlns:converters="clr-namespace:WalletWasabi.Gui.Converters;assembly=WalletWasabi.Gui"
+  x:Class="WalletWasabi.Gui.Controls.TransactionDetails.Views.TransactionDetails">
+  <UserControl.Resources>
+    <converters:BooleanStringConverter x:Key="BooleanStringConverter" />
```

Comment:
at somepoint we should just have a single static instance of these converters... and use them like x:State BooleanStringConverter.Instance

Reply:
I didn't know that it is possible but Concept ACK. Not in this PR.